### PR TITLE
move cob_map_accessibility_analysis to this repo

### PR DESCRIPTION
**Pull request by benmaidel**
removed it from `cob_scenario_states` https://github.com/ipa320/cob_scenario_states/pull/18

### CODE REVIEW

File: cob_map_accessibility_analysis/ros/include/cob_map_accessibility_analysis/map_accessibility_analysis_server.h [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+* and Automation (IPA) \n\n
+*
+*****************************************************************
+*
+* \note
+* Project name: care-o-bot
+* \note
+* ROS stack name: cob_scenario_states
+* \note
+* ROS package name: cob_generic_states_experimental
```
Comment (mgruhler) [inline cob_map_accessibility_analysis/ros/include/cob_map_accessibility_analysis/map_accessibility_analysis_server.h:17]: please adapt the names above :-)

File: cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_client.cpp [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+* and Automation (IPA) \n\n
+*
+*****************************************************************
+*
+* \note
+* Project name: care-o-bot
+* \note
+* ROS stack name: cob_scenario_states
+* \note
+* ROS package name: cob_generic_states_experimental
```
Comment (mgruhler) [inline cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_client.cpp:17]: as above

File: cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_server.cpp [hunk 3]
```diff
@@ -0,0 +1,709 @@
...
+* and Automation (IPA) \n\n
+*
+*****************************************************************
+*
+* \note
+* Project name: care-o-bot
+* \note
+* ROS stack name: cob_scenario_states
+* \note
+* ROS package name: cob_generic_states_experimental
```
Comment (mgruhler) [inline cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_server.cpp:17]: as above

**Comment by mgruhler** [comment]
While scrolling through, I noticed two things:
1. indentation is inconsistent between files, but consistent within (one cpp with tabs instead of spaces, another with 4 spaces, ...)
2. many `std::cout`. Should those be changed to the respective ROS_\* macros

@ipa-bnm what do you think?

**Comment by benmaidel** [comment]
@ipa-mig yep, but we should not solve this with this pr. This pr is just for moving the actual state of the package into this repo. Can you open a new Issue to keep track of this?

**Comment by mgruhler** [comment]
CR: ok

see #69 for further improvements

**Comment by NadiaHG** [comment]
CR:OK

**Pull request merged**